### PR TITLE
First and Second videos', text and vtt.

### PR DESCRIPTION
**Pull request by idilsezgin**
First and Second videos', text and vtt.

### CODE REVIEW

**Comment by egekorkan** [comment]
The PRs #44 and #45 seems to have been closed due to IPR checks. There is no need to close them, we can ask for a revalidation once the w3c linking issue is solved by the PR submitter. This PR looks good but we need a readme. 

Also, I have realized that we may need a folder where we merge and move to the final folder for publication to YouTube. Otherwise, people need to review the document while they are in the PR area, which might be annoying for some. It is a bit against git principles though... @relu91 any opinions?

Side note: It may actually increase the repository size if we put a file and move it since moving a file is not a git operation afaik. This means that we will have removal and addition of a _big_ file, thus a big .git folder size when cloning.

**Comment by jyhi** [comment]
> a folder where we merge and move to the final folder

Use a branch :)

**Comment by egekorkan** [comment]
@jyhi thanks for the opinions. I think that the second option is indeed better (not leaving the PR environment). For those who are reviewing, I propose to put comments on the vtt file if they have remarks for the text or for the video. That way, we know which seconds to look at even if it is feedback about the visuals.

File: Tutorials/whatiswot/1.WoT/1.WoT.vtt [hunk 1]
```diff
@@ -0,0 +1,110 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:04.889 
+[pause 5.5s]
+
+00:00:04.889 --> 00:00:10.293 
+Hello! Welcome to the "What is WoT" video series. [pause strong]
```
Comment (relu91) [inline Tutorials/whatiswot/1.WoT/1.WoT.vtt:7]: this `[]` is leaking in the captions subtitles. I am not sure if it is intended... but they distract a little bit.

**Review by relu91 — CHANGES_REQUESTED** [review]
I feel is better to remove the square bracket comments, but I am open to discussion. Minor points:
1. Default video reader from Windows 11 doesn't let you change caption color -> the subtitle is white on white 😢 switching to Film&Video gives you better readability. 
2. WoT.vtt is picked automatically, while `Payload-and-headers.vtt` no. It's weird. 

### reviewing
@egekorkan asked me to give a little bit of feedback on how to review this kind of content. I think that as long as we are dealing with additions the review is pretty straightforward. Were you clone the repo and switch to the right branch you can move freely and test everything. For sure you need to have a little expertise in working with Github.  I would say is going to be hard if you change just a slide in the PowerPoint... the PR author should somehow help the reviewers and guide them on the changes.

File: Tutorials/whatiswot/1.WoT/1.WoT.vtt (hunk 1 above)
Comment (egekorkan) [reply Tutorials/whatiswot/1.WoT/1.WoT.vtt:7]: @idilsezgin there is an option to exclude the pauses in murf. Could you reupload them with that?

**Comment by egekorkan** [comment]
> WoT.vtt is picked automatically, while Payload-and-headers.vtt no. It's weird.

There was a typo with the filenames (- vs .)

**Comment by idilsezgin** [comment]
@relu91, changes are made. Could you please take a look?

File: Tutorials/whatiswot/1.WoT/1.WoT.vtt (hunk 1 above)
Comment (idilsezgin) [reply Tutorials/whatiswot/1.WoT/1.WoT.vtt]: .vtt files are generated again without the brackets.

**Pull request merged**